test/test_csvs.py

```diff
@@ -0,0 +1,289 @@
+# OOIPLACEHOLDER
+#
+# Copyright 2018 Raytheon Co.
+
+import os
+import unittest
+from nose.plugins.attrib import attr
+
+@attr('UNIT')
+class GeneralUnitTest(unittest.TestCase):
+    
+    def _getPath(self, filename):
+        return os.path.join(os.path.dirname(__file__), filename)
+
+    def _getParameters(self):
+        parameters = {} # dictionary: key=name; value=data type number
+        with open(self._getPath('parameter.txt'), mode='rt') as f:
+            for line in f:
+                parts = line.strip().split(':')
+                parameters[parts[0]] = parts[1]
+        return parameters
+
+    def _getRefDesignators(self):
+        refdes = set()
+        with open(self._getPath('refdes.txt'), mode='rt') as f:
+            for line in f:
+                refdes.add(line.strip())
+        return refdes
+
+    def _getTypes(self):
+        types = {} # dictionary: key=number; value=data type
+        with open(self._getPath('type.txt'), mode='rt') as f:
+            for line in f:
+                parts = line.strip().split(':')
+                types[parts[0]] = parts[1]
+        return types
+
+    def _getUnits(self):
+        units = set()
+        units.add('') # empty value is valid
+        with open(self._getPath('unit.txt'), mode='rt') as f:
+            for line in f:
+                units.add(line.strip())
+        return units
+
+    def _isEmptyOrNumeric(self, value):
```

Comment:
What I did was not pythonic! I'll try what you suggest since I'm thinking either option would consider a leading minus sign and a single decimal point. I'll have to see what it does about empty-string. That may remain as an additional upfront check.